servers/src/main/java/tachyon/master/MasterSource.java

```diff
@@ -160,43 +196,99 @@ public MetricRegistry getMetricRegistry() {
     return mMetricRegistry;
   }
 
-  public void incCreateFileOps() {
-    mCreateFileOps.inc();
+  public void incCompleteFileOps() {
```

Comment:
should this and the next two methods take a long input argument for consistency with the rest of the methods